src/NvimClient.API/NvimClient.API.csproj

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
     <DocumentationFile>bin\Debug\net5.0\NvimClient.API.xml</DocumentationFile>
+    <RepositoryUrl>https://github.com/neovim/nvim.net</RepositoryUrl>
+    <Version>0.20.0</Version>
```

Comment:
Note I only added these here because this is the only part of the repo that gets published; I could also add these to the others, thoughts?

Reply:
could be confusing to have it in places where it's not actually needed/wanted.